internal/types/base.go

```diff
@@ -0,0 +1 @@
+package types
```

Comment:
Why this is remained?

Reply:
Humm :)) not sure why! I think the file could be deleted.